src/api.py

```diff
@@ -331,8 +371,12 @@ def index():
 
     return jsonify(data)
 
-@app.route("/calculate")
+@APP.route("/calculate")
 def calculate_investment():
+    """
+    !!! DEPRECATED !!!
+    Calculate investment
+    """
```

Comment:
Don't you feel that it might be worth considering adding a decorator for deprecated methods?

A good example would be https://deprecation.readthedocs.io/en/latest/

Or if we don't want to add an external package for a deprecation decorator, we could always create our own.
It may not be _very_ useful currently, but as the project grows, it'll definitely help making the code more maintainable.

Reply:
That's a really good idea!